util/cc2file.go.tmpl

```diff
@@ -0,0 +1,67 @@
+{{/*
+	This command sends your CC(s) code in a text file, rather than "plain" Discord messages, preserving Tabs, markdown, etc. 
+	You can especify more than one ID or trigger, and bot will generate one file per input parameter. If none are provided, output will be CC list
+	
+	Usage: `-cc2file <ID or Trigger> [ID or Trigger] [ID or Trigger]...`. For example: `-cc2file 10 "te st" 23`
+	Recommended trigger: Command trigger with trigger `cc2file`.
+	Configuration variables: None
+*/}}
+
+
+{{define "sendLongMessage"}}	{{/*INPUT: cslice channelIDToSend textToSend stringSeparator*/}}
+	{{$channelID:= index . 0}} {{$text:= index . 1}} {{$separator:= index . 2}}
+	{{$outStr:= ""}}
+
+	{{- range (split $text $separator)}}
+		{{- if gt (add (len $outStr) (len .)) 2000}}
+			{{- sendMessage $channelID $outStr}}
+			{{- $outStr = .}}
+		{{- else}}
+			{{- $outStr = (joinStr $separator $outStr .)}}
+		{{- end}}
```

Comment:
```suggestion
		{{- end -}}
```

Reply:
Thanks! I know `-` strips out newlines and whitespaces. But when use it only at `{{` and when use in both sides of `{{  }}`?